Scripts/script-MD2HTML.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: MD2HTML
+  version: -1
+name: MD2HTML
+script: |-
+  from markdown2 import Markdown
+
+  markdowner = Markdown(extras=["tables"])
+  file_entry = demisto.args().get('file_id', '')
+  file_path = demisto.getFilePath(file_entry)['path']
```

Comment:
Normally better to verify the answer is actual dict then .get the 'path' so it wont raise exception in case things didn't go as expected.